smartsim/_core/_install/redisaiBuilder.py

```diff
@@ -0,0 +1,283 @@
+            target_dir = self.package_path / package.name
+            package.retrieve(target_dir)
+            # Move actual contents to root of the expected location
+            actual_root = self._find_closest_object(target_dir, "include")
+            if actual_root and actual_root != target_dir:
+                logger.debug(
+                    (
+                        "Non-standard location found: \n",
+                        "{str(actual_root)} -> {str(target_dir)}",
+                    )
+                )
+                for file in actual_root.iterdir():
+                    file.rename(target_dir / file.name)
+
+    def run_command(self, cmd: t.Union[str, t.List[str]], cwd: pathlib.Path) -> None:
+        """Executor of commands usedi in the build
+
+        :param cmd: The actual command to execute
+        :param cwd: The working directory to execute in
+        """
+        stdout = None if self.verbose else subprocess.DEVNULL
+        stderr = None if self.verbose else subprocess.PIPE
+        proc = subprocess.run(
+            cmd, cwd=str(cwd), stdout=stdout, stderr=stderr, check=False
+        )
+        if proc.returncode != 0:
+            print(proc.stderr.decode("utf-8"))
+            raise RedisAIBuildError(
+                f"RedisAI build failed during command: {' '.join(cmd)}"
+            )
+
+    def _rai_cmake_cmd(self) -> t.List[str]:
+        """Build the CMake configuration command
+
+        :return: CMake command with correct options
+        """
+
+        def on_off(expression: bool) -> t.Literal["ON", "OFF"]:
+            return "ON" if expression else "OFF"
+
+        cmake_args = {
+            "BUILD_TF": on_off(self.build_tensorflow),
+            "BUILD_ORT": on_off(self.build_onnxruntime),
+            "BUILD_TORCH": on_off(self.build_torch),
+            "BUILD_TFLITE": "OFF",
+            "DEPS_PATH": str(self.package_path),
+            "DEVICE": "gpu" if self.platform.device.is_gpu() else "cpu",
+            "INSTALL_PATH": str(CONFIG.lib_path),
+            "CMAKE_C_COMPILER": self.build_env.CC,
+            "CMAKE_CXX_COMPILER": self.build_env.CXX,
+        }
+        if self.platform.device.is_rocm():
+            cmake_args["Torch_DIR"] = str(self.package_path / "libtorch")
+        cmd = ["cmake"]
+        cmd += (f"-D{key}={value}" for key, value in cmake_args.items())
+        cmd.append(str(self.src_path))
+        return cmd
+
+    @property
+    def _rai_build_cmd(self) -> t.List[str]:
+        """Shell command to build RedisAI and modules
+
+        With the CMake based install, very little needs to be done here.
+        "make install" is used to ensure that all resulting RedisAI backends
+        and their dependencies end up in the same location with the correct
+        RPATH if applicable.
+
+        :return: Command used to compile RedisAI and backends
+        """
+        return "make install -j VERBOSE=1".split(" ")
```

Comment:
Yeah, otherwise I think it's hard to determine whether we want to be verbose in not capturing output or make the builds noisy. Most of the time when we turn verbose on, we're troubleshooting so this force of `VERBOSE=1` effectively gives us the most output possible